utils/Posicao.java

```diff
@@ -0,0 +1,11 @@
+package utils;
+
+public class Posicao {
+    public int x;
+    public int y;
+
+    public Posicao(int x, int y) {
+        this.x = x;
```

Comment:
Essa posição você pensou em ser uma posição lógica variando nos inteiros de 0 a 7? Entendi certo? Não vejo problema em trabalhar no domínio dos inteiros, mas de qqr forma eventualmente vai ter que ter um mapeamento das colunas para [a...h] pela questão do histórico de jogadas por ex. Mas facilmente resolvivel ne kkkkk.

Reply:
Isso só pegar assim:
`'h'-this.y`
Que vai dar uma letra [a..h] equivalente